src/main/kotlin/no/nav/familie/ef/sak/simulering/SimuleringController.kt

```diff
@@ -31,4 +31,12 @@ class SimuleringController(
         tilgangService.validerTilgangTilBehandling(saksbehandling, AuditLoggerEvent.UPDATE)
         return Ressurs.success(simuleringService.simuler(saksbehandling))
     }
+
+    @GetMapping("/resultatstatus/{behandlingId}")
+    fun simuleringsresultatStatus(
```

Comment:
```suggestion
    fun erSimuleringsresultatEndret(
```

Ville jeg isåfall renamet denne til

Reply:
Lag gjerne en test på denne metoden